src/components/Principle.svelte

```diff
@@ -42,6 +43,14 @@
 
 <div class={className}>
   <Header>{principle.num}: {principle.handle}</Header>
+  <label for={`principle-applicability-${principle.num}`}>Applicable?</label>
+  <input 
+    type="checkbox" 
+    id={`principle-applicability-${principle.num}`} 
+    bind:checked={$evaluation['meta']['principleApplicability'][principle.num]} 
+    on:blur={() => {
+    evaluation.updateCache($evaluation);
```

Comment:
update the 'cache', this will updated our copy of the evaluation that exists in `localStorage`, we do this for any change anywhere

Reply:
At this point adding the <label for={`principle-applicability-${principle.num}`}> [...] <input> etc breaks my build.

Reply:
I get an Uncaught TypeError:

 Principle.svelte:41 Uncaught TypeError: Cannot set property 'Input.1' of undefined
    at HTMLInputElement.input_change_handler (Principle.svelte:41)

Reply:
It may be that you're still using an existing `evaluation` object that does not have this `principleApplicability` property.

You can clear this by running `localStorage.clear()` in the browser dev tools (or start a new evaluation button).